src/UI/examples/Button/Toggle/base.php

```diff
@@ -0,0 +1,37 @@
+<?php
+function base() {
+	global $DIC;
+	$factory = $DIC->ui()->factory();
+	$renderer = $DIC->ui()->renderer();
+
+	$message = 'Are you sure you want to delete the following items?';
+	$form_action = $DIC->ctrl()->getFormActionByClass('ilsystemstyledocumentationgui');
+	$icon = $factory->image()->standard('./templates/default/images/icon_crs.svg', '');
+	$modal = $factory->modal()->interruptive('My Title', $message, $form_action)
+		->withAffectedItems(array(
+			$factory->modal()->interruptiveItem(10, 'Course 1', $icon, 'Some description text'),
+			$factory->modal()->interruptiveItem(20, 'Course 2', $icon, 'Another description text'),
+			$factory->modal()->interruptiveItem(30, 'Course 3', $icon, 'Last but not least, a description'),
+		));
+
+	$modal2 = $factory->modal()->interruptive('Your Title', $message, $form_action)
+		->withAffectedItems(array(
+			$factory->modal()->interruptiveItem(40, 'Course 4', $icon, 'Some description text'),
+			$factory->modal()->interruptiveItem(50, 'Course 5', $icon, 'Another description text'),
+			$factory->modal()->interruptiveItem(60, 'Course 6', $icon, 'Last but not least, a description'),
+		));
```

Comment:
This is overly complex. Please simplify the modals by just displaying, "Switched to on" and "Switched to off". This makes this example a lot easier to grasp. The story here is not about icons, interruptive items, post processing or similar. It's about the $toggle. In the show_modal_on_button_click example of the interruptive modal this all makes sense. Here it does not.